index.html

```diff
@@ -155,6 +157,83 @@ <h2>
         </p>
       </div>
     </section>
+    <section> <h3>Permissions and user prompts</h3>
+      <p>
+        The term <a>obtain permission</a>
+        is used for acquiring trust for a given operation.
```

Comment:
A semantic nit: The browser or site doesn't "acquire" trust in a single action. Trust is harder to win than that, and needs to be earned over a series of interactions. I suspect this is the wrong term to use, and that you should try to stick to the terms in https://w3c.github.io/permissions/#requesting-more-permission.

Reply:
Yes, I copied this from Web NFC, so we probably need to review that later. I removed this for now as it is a duplicate